src/evox/algorithms/so/pso_vatients/cpso_s.py

```diff
@@ -0,0 +1,129 @@
+import jax
+import jax.numpy as jnp
+
+import evox as ex
+from evox.utils import *
+
+# SL-PSO: Social Learning PSO
+# SL-PSO-GS: Using Gaussian Sampling for Demonstator Choice
+# https://ieeexplore.ieee.org/document/6900227
+@ex.jit_class
+class CPSO_S(ex.Algorithm):
```

Comment:
Class name should use `CamelCase`, so it's "CPSOS", a little bit silly, but it is what it is.